amr-wind/wind_energy/actuator/disk/constant_ct_ops.H

```diff
@@ -0,0 +1,185 @@
+#ifndef CONSTANT_CT_OPS_H_
+#define CONSTANT_CT_OPS_H_
+
+#include "amr-wind/wind_energy/actuator/disk/ConstantCt.H"
+#include "amr-wind/wind_energy/actuator/actuator_ops.H"
+#include "amr-wind/utilities/tensor_ops.H"
+
+namespace amr_wind {
+namespace actuator {
+namespace ops {
+
+template <>
+struct ReadInputsOp<ConstantCt, ActSrcDisk>
+{
+    void operator()(ConstantCt::DataType& data, const utils::ActParser& pp)
+    {
+        auto& ddata = data.meta();
+        auto& info = data.info();
+        pp.get("num_force_points", ddata.num_force_pts);
+        // eventually we will change velocity sampling location
+        // pp.get("num_vel_points_r", ddata.num_vel_points_r);
+        // pp.get("num_vel_points_t", ddata.num_vel_points_t);
+        // pp.get("diameters_to_sample_velocity", ddata.diameters_to_sample);
+        pp.get("disk_center", ddata.center);
+        pp.get("disk_normal", ddata.normal_vec);
```

Comment:
Hi @psakievich, do you know if it would be possible to include provide turbine `yaw` as an input?  It could be in addition to `disk_normal`, with `yaw` overriding `disk_normal` in case both are there.  I'm thinking that most cases will assume turbine will be on a flat-ish terrain, and be dealing with turbine yaw from the farm layout inputs.

Reply:
yeah this sounds good

Reply:
@lawrenceccheung do you think we should also add a pitch option as well?

Reply:
Good thought.  If you're referring to turbine tilt, yes, we should probably add that in (I'd probably call it `tilt` or `shaft_tilt`, basically how much the rotor shaft deviates from horizontal, or how much the rotor disk deviates from vertical).  I don't think it's widely used in various modeling approaches yet, but since it can be a few degrees and noticeable on large turbines, might as well include it now.